test/e2e/lib/hooks-playwright/hooks.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable mocha/no-top-level-hooks */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+import { BrowserManager } from '@automattic/calypso-e2e';
+
+const playwrightTimeoutMS = config.get( 'playwrightTimeoutMS' );
+
+/**
+ * Hook to launch a new instance of BrowserContext then a new
+ * Page for testing.
+ *
+ * @returns {void} No return value.
+ */
+before( 'Launch browser instance', async function () {
+	this.browserContext = await BrowserManager.newBrowserContext();
+	await this.browserContext.setDefaultTimeout( playwrightTimeoutMS );
+	this.page = await this.browserContext.newPage();
+} );
+
+/**
+ * Hook to terminate a Browser instance.
+ *
+ * @returns {Promise<void>} Void promise.
+ */
+after( 'Close browser', async function () {
+	return await BrowserManager.browser.close();
```

Comment:
Nit: Mocha won't do anything with the returned value of an async hook, so no need to return anything.
```suggestion
    await BrowserManager.browser.close();
```

Reply:
Got it, in that case I should also update the return value in the JSDoc as well.